tests/integration/components/freestyle-showdown-content/component-test.js

```diff
@@ -0,0 +1,19 @@
+import { render } from '@ember/test-helpers';
```

Comment:
Added a simple render test to ensure that `ember-cli-showdown` still works properly.